docs/EN/CompatiblePumps/OmnipodDASH.md

```diff
@@ -66,21 +66,22 @@ Instead a temporary zero basal rate can be achieved through the "Disconnect pump
 
 ## Selecting Dash in AAPS
 
-There are **two ways**:
+There are **two** available options to configure Omnipod in **AAPS**:
 
 ### Option 1: New installations
 
-When installing **AAPS** for the first time, the **Setup Wizard** will guide new users through key features and installation requirements for **AAPS**. Select “DASH” when you reach Pump selection.
+When installing **AAPS** for the first time, the **Setup Wizard** will guide new users through key features and installation requirements for **AAPS**.  
+Select “DASH” when you reach Pump selection.
 
 ![Enable_Dash_1](../images/DASH_images/Enable_Dash/Enable_Dash_1.png)
 
-When in doubt you can also select “Virtual Pump” and select “DASH” later, after setting up **AAPS** (see option 2).
+When in doubt you can also select “Virtual Pump” and select “DASH” later, after setting up **AAPS** (See [Option 2](#Option-2:-The-Config-Builder)).
```

Comment:
Mind the format #option-2-the-config-builder, same as before: better an achor for this call, I added (omnipod-dash-option-2-config-builder)